### DISCUSSION TITLE
Album art problems

### DISCUSSION THREAD

**Discussion by jojo2357** [jojo2357/Music-Discord-Rich-Presence#59] [Album Art Problems]
For album art related questions, reply here with a copy of the MDRP terminal, a screenshot of what you expected, and what showed up

**Comment by SaqinNoor** [comment]
I know I'm probably being dumb, but how to fix this? As in how to key albums?
![image](https://user-images.githubusercontent.com/89857833/197234063-a1cb851e-bc88-4510-9af2-313b2ccf6d3b.png)

**Reply by jojo2357** [reply]
You can [read the readme](https://github.com/jojo2357/Music-Discord-Rich-Presence#custom-album-art), watch one of the video tutorials I made: [1.5](https://www.youtube.com/watch?v=_qkxjf3T8Pw) [1.6](https://youtu.be/GucN2WiteOM) and/or [1.7](https://www.youtube.com/watch?v=mhgKXxdhYoA). The videos all focus on different things, there really isnt much that doesnt apply anymore.

Alternatively, you could [enable remote art](https://github.com/jojo2357/Music-Discord-Rich-Presence#the-lazy-way) which will attempt to do it all for you.

**Comment by grayshirk** [comment]
![image](https://github.com/jojo2357/Music-Discord-Rich-Presence/assets/18224977/66d8f31f-0832-4d30-ad75-7e5352241676)
I have the album manually keyed-

**Reply by Smaltin** [reply]
As documented [here](https://github.com/jojo2357/Music-Discord-Rich-Presence/issues/76), the issue lies within Windows Music Player not properly feeding out data to WMC.
It was noted that opening the equalizer solved the issue for the OP, maybe that will help you?
